tests/components/samsungtv/test_config_flow.py

```diff
@@ -288,57 +298,67 @@ async def test_user_not_successful_2(hass: HomeAssistant, remotews: Mock):
 async def test_ssdp(hass: HomeAssistant, remote: Mock):
     """Test starting a flow from discovery."""
 
-    # confirm to add the entry
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": config_entries.SOURCE_SSDP}, data=MOCK_SSDP_DATA
-    )
-    assert result["type"] == "form"
-    assert result["step_id"] == "confirm"
+    with patch(
+        "homeassistant.components.samsungtv.bridge.SamsungTVWSBridge.device_info",
+        return_value=MOCK_DEVICE_INFO,
+    ), patch("getmac.get_mac_address", return_value="aa:bb:cc:dd:ee:ff"):
```

Comment:
We have this fixture already. We can use it and just change the return value.

https://github.com/home-assistant/core/blob/be94ce42a5447cbc018fe120e01cf865e114c2a2/tests/components/samsungtv/conftest.py#L117-L121